tests/performance-tests/atala-performance-tests-k6/README.md

```diff
@@ -29,3 +32,27 @@ Once that is done, we can run our script the same way we usually do, for instanc
 ```bash
 $ k6 run dist/connection-flow-test.js
 ```
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+# Debugging Tests
+=======
+## Debugging Tests
+>>>>>>> 7aa9b4c2 (feat: upgrade ZIO http client to improve performance (#850))
+=======
+## Debugging Tests
+>>>>>>> 7aa9b4c2 (feat: upgrade ZIO http client to improve performance (#850))
+
+k6 can be configured to log the HTTP request and responses that it makes during test execution. This is useful to debug errors that happen in tests when logs or k6 output does not contain the reason for a failure.
```

Comment:
k6 is configurable to log the HTTP request and responses during test execution. This is useful for debugging errors that occur in tests when the reason for a failure is not included in the logs or k6 output.